draft-irtf-cfrg-frost.md

```diff
@@ -826,17 +830,16 @@ Where Signature.R_encoded is `G.SerializeElement(R)` and Signature.z_encoded is
 
 The Coordinator SHOULD verify this signature using the group public key before publishing or
 releasing the signature. Signature verification is as specified for the corresponding
-ciphersuite; see {{ciphersuites}} for details.
-
-The aggregate signature will verify successfully if and only if all signature shares are valid.
-In other words, if there exists an invalid signature share, then the resulting aggregate
-signature will not verify successfully against the group public key. Moreover, subsets of
-valid signature shares will themselves not yield a valid aggregate signature.
+ciphersuite; see {{ciphersuites}} for details. The aggregate signature will verify successfully
+if and only if all signature shares are valid. In other words, if there exists an invalid
+signature share, then the resulting aggregate signature will not verify successfully against
+the group public key. Moreover, subsets of valid signature shares will themselves not yield
+a valid aggregate signature.
```

Comment:
note: I don't think "if and only if" holds. If the shares are valid, then the aggregate signature will be valid but I don't think the other direction holds.

Reply:
One share is off by -1, yet another is off by +1. Each share is invalid yet the aggregate will be fully valid. There's also random chance for two invalid, randomly derived shares, to collide.